lib/tracing-limit/Cargo.toml

```diff
@@ -15,6 +15,7 @@ dashmap = "4"
 [dev-dependencies]
 criterion = "0.3"
 tracing = "0.1.15"
+mock_instant = { version = "0.2" }
```

Comment:
Seemingly not a very popular crate, but I took a look and it seems to do the trick and is fairly stable.

https://github.com/museun/mock_instant